blueprints/evolutionapi/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - LANGUAGE=${LANGUAGE}
       - CONFIG_SESSION_PHONE_CLIENT=${CONFIG_SESSION_PHONE_CLIENT}
       - CONFIG_SESSION_PHONE_NAME=${CONFIG_SESSION_PHONE_NAME}
+      - CONFIG_SESSION_PHONE_VERSION=${CONFIG_SESSION_PHONE_VERSION}
```

Comment:
```suggestion
```

The variable `CONFIG_SESSION_PHONE_VERSION` [no longer exists in the project](https://github.com/search?q=repo%3AEvolutionAPI%2Fevolution-api%20CONFIG_SESSION_PHONE_VERSION%20&type=code).

The value used internally started being automatically populated in [version v2.3.0](https://github.com/EvolutionAPI/evolution-api/releases/tag/2.3.0).

It was completely removed in [version v2.3.1](https://github.com/EvolutionAPI/evolution-api/releases/tag/2.3.1).